package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "bodhi-api",
-  "version": "0.0.1",
+  "version": "0.2.0",
   "description": "An open source blockchain explorer for monitoring Bodhi platform",
   "main": "index.js",
   "scripts": {
-    "start": "babel-node --inspect index.js --presets es2015,stage-2",
+    "start": "babel-node --inspect index.js",
```

Comment:
I did this in docker branch. Could we make this two 
"start": "babel-node index.js",
"start:debug": "babel-node --inspect index.js",

so `npm start` will start with no debugger, and `npm run start:debug` will

Reply:
done